lib/Files.js

```diff
@@ -405,12 +405,15 @@ class Files {
   /* **************************** PUBLIC API METHODS ***************************** */
   /**
    *
-   * Lists files in a remote folder. If called on a file returns the file info if the file exists.
-   * If the file or folder does not exist returns an empty array.
+   * Lists files in a remote folder.
+   * A folder has trailing '/', otherwise this is a call on a file and will return the file info.
+   * If the path doesn't exists list returns an empty array.
```

Comment:
"If the path doesn't exist, the list returns an empty array."

Reply:
We might want to be clearer here that a path without a trailing slash is ALWAYS evaluated as a file, and if that file does not exist, we return [ ] even if there IS a folder with that same name.